src/borg/archive.py

```diff
@@ -1988,6 +2006,7 @@ def save(self, archive, target, comment=None, replace_original=True):
             archive.delete(Statistics(), progress=self.progress)
             target.rename(archive.name)
         if self.stats:
+            target.start = _start
             target.end = datetime.utcnow()
```

Comment:
so, if --timestamp is used, we'll have the desired timestamp as start value, but the current time as end value?

this might look like a backup taking a year, if the start time is a year in the past...

use case: having a set of old backups and re-creating them with borg.

Reply:
guess there could be 2 solutions for this:
- use start + duration as end time (this might be a bit fake though as the duration comes from the recreate operation, not from the create operation)
- use end == start (indicating "we don't know the end time" somehow)

Reply:
Oh, it highlighted an edge case I forgot indeed.
I think that we can give the statistics of the recreate command: how much time did it take to recreate the archive?